django/cantusdb_project/templates/flatpages/default.html

```diff
@@ -32,16 +32,29 @@ <h2>{{ flatpage.title }}</h2>
                     }
                 </script>
             </div>
-            <div class="card mt-3 w-100">
-                <div class="card-header">
-                    What's New
-                </div>
-                <div class="card-body">
-                    <ul>
+
+            {% if request.user.is_anonymous %}
+                <div class="card mt-3 w-100">
+                    <div class="card-header">
+                        What's New
+                    </div>
+                    <div class="card-body">
                         {% recent_articles %}
-                    </ul>
+                    </div>
                 </div>
-            </div>
+            {% endif %}
+
+            {% if not request.user.is_anonymous %}
```

Comment:
Django template language supports `{% if ... %}`, `{% else %}`, `{% endif %}`. That could make the code cleaner.

Reply:
very good - done!